verifier/src/lib.rs

```diff
@@ -168,22 +185,111 @@ pub enum VerificationError {
     },
 }
 
-impl DisplayableError for VerificationError {}
+impl ResultMessage for VerificationResultMetadata {
+    fn fmt_success(&self, f: &mut Formatter, pad: usize) -> core::fmt::Result {
+        let status = SUCCESS_MESSAGE_INDICATOR;
+        match self {
+            Self::General => write!(f, "{:pad$}{status} Success", "")?,
```

Comment:
Ideally it feels like the checkbox should be done by the caller, but then it gets weird rendering nested elements since the first line would ignore pad, but the subsequent lines would need pad.
```text
   Caller
      |
      v
    |----| 
    - [ ]  <main callee text>
|-Pad-|<subsequent line of callee>
```
While not ideal i think having each element write their checkbox makes the interface simpler